packages/studio-plugin/src/sourcefiles/ComponentFile.ts

```diff
@@ -42,12 +45,27 @@ export default class ComponentFile {
     };
 
     const filepath = this.studioSourceFileParser.getFilepath();
-
+    const cssImports = getCssFilesFromDependencyTree(this.dependencyTree);
     return {
       kind: FileMetadataKind.Component,
       ...this.fileMetadataParser.parse(onProp),
       ...(acceptsChildren ? { acceptsChildren } : {}),
       filepath,
+      cssImports,
     };
   };
 }
+
+function getCssFilesFromDependencyTree(dependencyTree: Tree): string[] {
+  let cssFiles: Set<string> = new Set();
+  Object.entries(dependencyTree).forEach(([filename, subDependencyTree]) => {
```

Comment:
if we use `reduce` here instead of `forEach`, I think that should simplify it a little and we should be able to remove the `let`

Reply:
I gave it a shot, and I think since I'm using a `Set` to avoid repetition, it won't work since its not iterable in the same way arrays are, interestingly.

I could do some stuff with arrays, but I think that would make the code a little messy.  Are you OK with how it is now?

Reply:
Ohh I see-- yeah that works.  I did the same thing, except instead of ` new Set<string>()` as the initial value, I did ` {} as Set<string>` which broke the site.  Changed!